python/rmm/tests/test_rmm.py

```diff
@@ -765,6 +772,20 @@ def test_dev_buf_circle_ref_dealloc():
     gc.collect()
```

Comment:
I think the comments above this line are either outdated, or they are attempting to describe what would happen if we hadn't decorated `DeviceBuffer` with `no_gc_clear`, is that right?

Reply:
Correct - I fixed it so that it explains what is happening now instead